knowledge_repo/app/deploy/gunicorn.py

```diff
@@ -21,12 +21,21 @@ def __init__(self, *args, **kwargs):
         BaseApplication.__init__(self)
 
     def load_config(self):
-        options = {
+        args = self.cfg.parser().parse_args(self.cfg.get_cmd_args_from_env())
+
+        # Load up environment configuration.
+        for key, value in vars(args).items():
+            if key != 'args' and value is not None:
+                self.cfg.set(key, value)
+
+        # Lastly, update the configuration with any command line settings.
+        args = {
```

Comment:
@john-bodley for readability and future-proofing, let's have only use `args` once. e.g. above `env_args` and here `command_line_args`

Reply:
👍